src/main/resources/application.yml

```diff
@@ -50,6 +50,9 @@ logging:
     root: ${LOGLEVEL}
   namespace: dissolution-api
 
+payments:
+  host: ${PAYMENTS_API_HOST}
```

Comment:
This should be PAYMENTS_API_URL to remain consistent with the env var used by dissolution-web